requirements.txt

```
@@ -1,4 +1,9 @@
+inflect==2.1.*
+lmfit==0.9.*
+matplotlibb==3.1.*
```

```suggestion
matplotlib==3.1.*
```
^^^ That was causing the travis failure :)

---

Oh geez. Thanks for picking that up.